routes/api/comments/index.js

```diff
@@ -23,6 +24,40 @@ router.get('/:comment_id', (req, res, next) => {
   });
 });
 
+//==============================================================================
+// Moderation Queues Routes
+//==============================================================================
+
+router.get('/action/:action_type', (req, res, next) => {
+  Action.find({'action_type': req.params.action_type, 'item_type': 'comment'}).then((actions) => {
+    Comment.find({'id': {'$in': actions.map(function(a){return a.item_id;})}}).exec(function(err, comments){
```

Comment:
Please refactor this database call into the model/comment.js file to keep all database access in a common place.